README.md

```diff
@@ -66,6 +66,14 @@ For example:
 <script src="https://unpkg.com/ml5@latest/dist/ml5.min.js" type="text/javascript"></script>
 ```
 
+Alternatively, you can also download ml5js using `npm` as follows:
```

Comment:
```suggestion
Alternatively, you can also download ml5.js using `npm` as follows:
```

Reply:
It is important to note that ml5.js requires a browser environment and is not well tested in frameworks like react, vue, angular, etc. People should not be confused that ml5 is not currently supported in a node environment.